rust/connlib/.gitignore

```diff
@@ -0,0 +1,176 @@
+### Rust ###
+target/
```

Comment:
This is outdated because the `target` directory will be one level above.

Reply:
:+1:  haven't fixed `.gitignore` yet, I just copy pasted it from the old repo

Reply:
I figured as much, just left the comment so it is tracked :)